src/dbusadaptor.h

```diff
@@ -25,6 +25,7 @@ class DBusAdaptor : public QDBusAbstractAdaptor
 
 public slots:
     void openUrl(QStringList args);
+    void openNewTab();
```

Comment:
I think this should be something like enterNewTabUrl() or activateNewTabView() as this doesn't yet open new tab instead just brings up the view where user can enter the new tab url or activate a favorite.

I'd like to see openNewTab(QString url) reversed for actually opening an external link to a new tab without activating the overlay.